lib/templates/_head.html

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="generator" content="dartdoc {{version}}">
```

Comment:
Can you add the "Made with <3" back? I like the friendly tone.

Otherwise, thanks for the upgrade to generator!